### ISSUE TITLE
I can't reproduce the results of the paper

### ISSUE THREAD

**Issue by Mowenyii** [SalesforceAIResearch/DiffusionDPO#11]
Thank you for your impressive work.

I can't reproduce the results of DPO-SD 1.5.

We train on 8 NVIDIA A100 GPUs with a local batch size of 1 pair and gradient accumulation of 256 steps. Other experimental Settings are the same as those in the paper.

Here are some of the results I sampled during the training.
<img width="440" alt="image" src="https://github.com/SalesforceAIResearch/DiffusionDPO/assets/50766241/de3701c4-231a-4df7-921d-3c3833af7a58">

There is also something strange about the loss function during training. The training process took about nine hours.
<img width="545" alt="image" src="https://github.com/SalesforceAIResearch/DiffusionDPO/assets/50766241/e239a77c-67b2-46a4-a032-ec97408c0515">

Can you give me some advice?

**Comment by DwanZhang-AI** [comment]
Hi, Why local batch size should be 1? A bigger local batch size may cause a performance degradation?

**Comment by lucasxu777** [comment]
Hi, did you solve this issue?

**Comment by SuperKenVery** [comment]
Same here. 

@DwanZhang-AI Did you see different performance between batch size=1 and >1?
